src/shared/components/ncTable/mixins/permissionsMixin.js

```diff
@@ -2,6 +2,9 @@ import { getCurrentUser } from '@nextcloud/auth'
 
 export default {
 	methods: {
+		ownsTable(element) {
+			return element?.ownership === getCurrentUser().uid
```

Comment:
Just from reading the code i think this should be:

```suggestion
			return element?.owner === getCurrentUser().uid
```

Reply:
I double cheched and it's `ownership`

Reply:
All right, thanks for double checking